lib/MusicBrainz/Server/Edit/Role/IPI.pm

```diff
@@ -12,6 +23,62 @@ with 'MusicBrainz::Server::Edit::Role::ValueSet' => {
     extract_value => sub { shift->ipi }
 };
 
+after initialize => sub {
+    my ($self, %opts) = @_;
+
+    my $old_ipis = $self->data->{old}{ipi_codes};
+    my $new_ipis = $self->data->{new}{ipi_codes};
+    my $added_ipis_set = Set::Scalar->new(@$new_ipis) - Set::Scalar->new(@$old_ipis);
+    my @added_ipis = $added_ipis_set->members;
+    $self->reused_ipis($self->c->model($self->_edit_model)->find_reused_ipis(@added_ipis));
+};
+
+after post_insert => sub {
+    my $self = shift;
+
+    if ($self->reused_ipis) {
+        for my $ipi (keys %{ $self->reused_ipis }) {
+            my $artist_dupe_count = $self->reused_ipis->{$ipi}->{artist};
+            my $label_dupe_count = $self->reused_ipis->{$ipi}->{label};
+            my $edit_note;
+            if ($artist_dupe_count && $label_dupe_count) {
+                 $edit_note = localized_note(N_l('The IPI {ipi} is already in use on both artists and labels ({artist_count} and {label_count} at the time of this note respectively). You can search for uses of this IPI on {artist_search|artists} and {label_search|labels}.'),
+                    {
+                        artist_count => $artist_dupe_count,
+                        artist_search => "/search?query=ipi%3A$ipi&advanced=1&type=artist",
+                        ipi => $ipi,
+                        label_count => $label_dupe_count,
+                        label_search => "/search?query=ipi%3A$ipi&advanced=1&type=label",
+                    }
+                );
+            } elsif ($artist_dupe_count) {
+                $edit_note = localized_note(N_l('The IPI {ipi} is already in use on artists ({artist_count} at the time of this note). You can {artist_search|search for uses of this IPI}.'),
```

Comment:
Using `N_ln` would allow for a more fluent phrase with “in use on another artist” and “in use on {n} other artists”.

I would suggest to make the second phrase more pressing, e.g. “Please check {artist_search|uses of this IPI}.”

Reply:
localized_note doesn't support N_ln, that's why this is done that way.

Reply:
Adding support seems feasible and worth it since more edit notes are being coded. Reported it under MBS-11171. That should not stop this PR though.

Reply:
I'd probably rather just add support for `ln` here, since these strings sound kind of awkward reading them again. I can help with that.

Reply:
I understood the ln support was more complex than expected?